schemas/asset_schema.json

```diff
@@ -182,7 +182,7 @@
                                 },
                                 "additionalAxles": {
                                     "type": "array",
-                                    "description": "An optional array of additional axles with the same properties as frontAxle and rearAxle.",
+                                    "description": "An optional array of additional axles with the same properties as 'frontAxle' and 'rearAxle'.",
```

Comment:
Wrong style of quotation mark? Also all of the others in this file.

Reply:
You implemented that style of quotation marks for examples in the material files and as we have no special mark-up for that I thought to leave it like that, or do you prefer using ""?

Reply:
Ah okay. I thought you wanted to use the typography style. But then its fine.